File: platforms/python/examples/scene/model/set_visible.py
from fragmentcolor import Material, Mesh, Model, Vertex

mesh = Mesh()
mesh.add_vertex(Vertex.pbr([0.0, 0.5, 0.0]))
blob = Model(mesh, Material.pbr())

# Wide zoom level — skip the detail blobs.
blob.set_visible(false)
# Zoom back in — turn them on again.
blob.set_visible(true)

File: platforms/swift/examples/scene/model/set_visible.swift
import FragmentColor

let mesh = Mesh()
try mesh.addVertex(Vertex.pbr([0.0, 0.5, 0.0]))
let blob = Model(mesh, Material.pbr())

// Wide zoom level — skip the detail blobs.
blob.setVisible(false)
// Zoom back in — turn them on again.
blob.setVisible(true)